app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class BooksController < ApiController
+      include Wor::Paginate
+      include Brita
```

Comment:
Think of this going forward with more controllers, you'd have to include this in every controller... What about putting these includes in the ApiController?